astroquery/jplhorizons/tests/test_jplhorizons_remote.py

```diff
@@ -222,7 +222,7 @@ def test_elements_query_raw(self):
                                    epochs=2451544.5).elements(
                                        get_raw_response=True)
 
-        assert len(res) >= 7400
+        assert len(res) >= 6686
```

Comment:
why do these numbers change? (totally just out of curiosity, I saw that these are tests that have been failing for a while independent of the API change)

Reply:
OK fair enough. If this breaks again, and the exact number doesn't really matter, then we can cook up a more robust check.